admin/src/js/controllers/edit-user.js

```diff
@@ -232,6 +236,27 @@ angular
       return true;
     };
 
+    const validatePlacesPermission = () => {
+      const userRoles = $scope.editUserModel.roles;
+
+      if (!$scope.editUserModel.place || $scope.editUserModel.place.length <= 1) {
+        return true;
+      }
+
+      const allowedRoles = $scope.permissions.can_have_multiple_places;
+
+      const userHasPermission = userRoles.some(role => allowedRoles.includes(role));
```

Comment:
I think we're duplicating logic that we already have in the cht script api. This should already be available in the admin app. https://github.com/medic/cht-core/blob/master/admin/src/js/services/auth.js#L65

Reply:
Hmm, the value of `userCtx.roles` is admin. It ends up being a check whether current user (Admin) has the permission

Reply:
Yea, I think you have to call it like: 
```
return chtScriptApi.v1.hasPermissions(['can_have_multiple_places'], $scope.editUserModel.role, settings.permissions);
```